astropy/wcs/tests/test_wcs.py

```diff
@@ -13,19 +13,22 @@
 
 from astropy.tests.helper import raises, catch_warnings
 from astropy import wcs
-from astropy.wcs import _wcs
+from astropy.wcs import _wcs  # noqa
```

Comment:
`flake8` complains that this import is not used but if I remove it, tests fail. :woman_shrugging: